src/main/resources/assets/createcasing/lang/ru_ru.json

```
@@ -237,20 +425,28 @@
   "createcasing.ponder.depot.text_2": "Кликните ПКМ, чтобы самостоятельно положить или забрать предметы с депо",
   "createcasing.ponder.depot.text_3": "Так же как и конвейер, депо может предоставлять предметы для обработки",
   "createcasing.ponder.depot.text_4": "А также поставлять предметы механическим рукам",
-  
+
   "createcasing.ponder.chain_drive.header": "Передачей силы вращения с помощью цепных приводов в корпусе",
   "createcasing.ponder.chain_drive.text_1": "Цепные приводы передают силу вращения друг другу",
   "createcasing.ponder.chain_drive.text_2": "Таким образом, все соединённые валы будут вращаться в одном направлении",
   "createcasing.ponder.chain_drive.text_3": "Любая часть в ряду может быть повёрнута на 90 градусов",
-  
+
   "createcasing.ponder.chain_gearshift.header": "Использованием регулируемых цепных коробок передач",
   "createcasing.ponder.chain_gearshift.text_1": "Неактивные регулируемые цепные коробки передач ведут себя точно так же, как цепные приводы",
   "createcasing.ponder.chain_gearshift.text_2": "При активации, скорость передаваемая другим цепным коробкам передач в ряду удваивается",
   "createcasing.ponder.chain_gearshift.text_3": "Когда активированная регулируемая цепная коробка передач не является источником, её скорость будет снижена вдвое",
   "createcasing.ponder.chain_gearshift.text_4": "В обоих случаях, цепные приводы в корпусе всегда вращаются с удвоенной скоростью активированной регулируемой цепной коробки передач",
   "createcasing.ponder.chain_gearshift.text_5": "Используя аналоговые редстоун-сигналы, изменение скорости может быть настроено более точно, нежели в 1 или в 2 раза",
   "createcasing.ponder.chain_gearshift.text_6": "12 об/мин",
-  
+
+  "createcasing.ponder.configurable_gearbox.header": "Коробки передач такие скучные!",
```

Честно говоря, не знаю, чем тут руководствовался Iglee, ибо в хэдерах размышлений чётко пишется, о чём игрок "размышлает". Потому в русском переводе есть эмпирическое правило: любой `ponder.header` - в **винительном падеже**.
Перевод не всегда должен быть буквальным. Я бы предложил вариант "Нескучной альтернативой коробки передач" **[В.п.]**, чтобы сохранить эмоциональный окрас, но соблюсти все правила :D

---

Имеется в виду такой вариант?
```suggestion
  "createcasing.ponder.configurable_gearbox.header": "Нескучной альтернативой коробки передач",
```

---

> Имеется в виду такой вариант?

Именно! Только исправьте `коробкИе`, пожалуйста)